### PR TITLE
Edits to Using-Radicle/Overview

### PR DESCRIPTION
**Pull request by abbey-titcomb**
Cleaned up Overview page.

### CODE REVIEW

File: docs/using-radicle/overview.md [hunk 1]
```diff
@@ -5,28 +5,28 @@ title: Overview
...
+no single canonical view (e.g. master) of any project, but multiple upstreams
 maintained by maintainers and contributors that exchange patches with each
 other.
 
 Within the same project, two people will have subjective (and often diverging
 views), but *your* view of the project becomes the sum of all other views of the
-project. This means that when you find and follow a project on Radicle, you are
-not only able to replicate the maintainer's view of the project, but all other
+project. This means that when you find and follow a project on Radicle, you
+don't just replicate the maintainer's view of the project, but all other
```
Comment (xla) [inline docs/using-radicle/overview.md:16]: I don't this is true. AFAIK there is no implicit replication, every peers view has to be explicitly tracked. We should confirm.

**Review by xla — APPROVED** [review]
🐖 🌉 🌘 🕗

File: docs/using-radicle/overview.md (hunk 1 above)
Comment (abbey-titcomb) [reply docs/using-radicle/overview.md:16]: I thought there was... I'll remove for now to unblock this PR.

File: docs/using-radicle/overview.md [hunk 2]
```diff
@@ -5,28 +5,28 @@ title: Overview
...
 contributor's views as well. Conceptually, the project becomes decentralized
```
Comment (abbey-titcomb) [inline docs/using-radicle/overview.md:17]:
```suggestion
project. Conceptually, the project becomes decentralized
```

**Review by xla — APPROVED** [review]
💷 🚣 🚔 🛩

**Pull request merged**